org.serenityos.Ladybird-gtk4.desktop

```diff
@@ -17,13 +18,15 @@ X-Purism-FormFactor=Workstation;Mobile;
 [Desktop Action new-window]
 Name=New Window
 Name[eo]=Nova fenestro
+Name[fr]=Nouvelle Fenêtre
 Name[ru]=Новое окно
 Name[sv]=Nytt fönster
 Exec=ladybird --no-such-option
 
 [Desktop Action new-incognito-window]
 Name=New Incognito Window
 Name[eo]=Nova inkognita fenestro
+Name[fr]= Nouvelle Fenêtre de Navigation Privée
```

Comment:
So when originally deciding between "private mode" or "incognito", I went for incognito, mostly because I didn't want to make it feel that the default experience is un-private (since Ladybird won't spy on you whether you're in incognito mode or not). If there's not a good French translation of "incognito", it's totally fine to call it "private navigation" (which is what that means, I assume). But if there is, please try to use that wording 🙂

Reply:
Incognito is a valid French word: it means "secret", as in not to be seen.